### ISSUE TITLE
[iOS] Grayscale and Sepia filters/components don't work with React Native v0.60.0

### ISSUE THREAD

**Issue by RohanTalip** [iyegoroff/react-native-color-matrix-image-filters#20]
```
$ react-native --version
react-native-cli: 2.0.1
react-native: 0.60.0
```
```
$ yarn list react-native react-native-color-matrix-image-filters
yarn list v1.16.0
warning Filtering by arguments is deprecated. Please use the pattern option instead.
├─ react-native-color-matrix-image-filters@5.1.5
└─ react-native@0.60.0
✨  Done in 0.53s.
```

The Grayscale and Sepia components work as expected with React Native v0.59.8 but not with React Native v0.60.0; in React Native version 0.60.0 when I wrap an image with the Grayscale or Sepia components, it just disappears completely.

I even linked the libRNColorMatrixImageFilters.a library in Xcode's Build Phases tab.

I had to apply PR #19 to even get the project to build in Xcode though.

The iOS app was a barebones one created via `react-native init MyAppName` for RN v0.60.0, and via `react-native init --version="react-native@0.59.8" MyAppName0598` for RN v0.59.8.  For RN v0.59.8 I also had to run "pod install" in the ios directory and change the Podfile to get the project to build as expected.

I then added an Image component with a source of a colour image to verify that the image could be loaded.  Then I wrapped the Image component with both Grayscale and Sepia components/filters, i.e. identical code for both versions of React Native.  As mentioned earlier, it worked in React Native version 0.59.8 but the image disappeared in React Native version 0.60.0 (with either Grayscale or Sepia wrapping it).

Nothing relevant appears in the console logs in Xcode.

Other filters/components from react-native-color-matrix-image-filters might or might not work; I didn't try them.

Do you have any ideas as to either what I'm doing wrong or what code or configuration needs to change?

**Comment by iyegoroff** [comment]
Fixed in v5.1.6

**Comment by RohanTalip** [comment]
Thanks for the quick fix!
